maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/repository/ApplicantBillingRepository.java

```diff
@@ -25,4 +27,15 @@ public interface ApplicantBillingRepository extends JpaRepository<ApplicantBilli
             """,
             nativeQuery = true)
     List<ApplicantBillingEntity> findAllApplicantsForBilling();
+
+    @Modifying
```

Comment:
Instead of taking the ids as input, we would need to join on the MAAT_REFS_TO_EXTRACT table.

Reply:
After a chat with Karl yesterday, he mentioned that we won't be able to do the join to determine the ids for the flag reset, as the maat_refs_to_extract table doesn't contain all of the data that will be sent due to the UNIONs in the extracts SQL.